pom.xml

```diff
@@ -1034,6 +974,47 @@
           <artifactId>templating-maven-plugin</artifactId>
           <version>1.0.0</version>
         </plugin>
+        <!-- REST Documentation plugin -->
+        <plugin>
+          <groupId>com.github.kongchen</groupId>
+          <artifactId>swagger-maven-plugin</artifactId>
+          <version>3.1.1</version>
+          <configuration>
+            <apiSources>
+              <apiSource>
+                <springmvc>false</springmvc>
+                <schemes>http,https</schemes>
+                <host>[Alluxio Master Hostname]</host>
+                <basePath>/api/v1</basePath>
+                <info>
+                  <title>Alluxio REST API Documentation</title>
+                  <version>v1</version>
+                  <termsOfService>
+                    http://www.github.com/alluxio/alluxio
+                  </termsOfService>
+                  <description>Documentation for Alluxio RESTful APIs</description>
+                  <contact>
+                    <email>haoyuan@alluxio.com</email>
```

Comment:
this email is used for the "Contact the developer" link. i'm guessing we shouldn't use HY's email here but i'm not sure which email alias would be best given that this is going into open source...

Reply:
I used the same email/name associated with the Alluxio repository, but another option is just to remove this section.

Reply:
I've gotten rid of this.